### PR TITLE
Feature: Add Appointment remark

### LINKED ISSUE #139: As a user, I can add remarks for appointment
So that I can keep track of any significant finding or event during the medical appointment.

### PR DESCRIPTION
**Pull request by mounilsankar**
Closes #139

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/AppointmentRemarkCommand.java [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Edits the remark of the appointment identified "
+            + "by the index number used in the last appointment listing. "
+            + "Existing remark will be overwritten by the input.\n"
+            + "Parameters: INDEX (must be a positive integer) "
+            + "r/ [REMARK]\n"
+            + "Example: " + COMMAND_WORD + " 1 "
+            + "r/ Likes to swim.";
+
+    public static final String MESSAGE_ADD_REMARK_SUCCESS = "Added remark to Person: %1$s";
```
Comment (cheeggered) [inline src/main/java/seedu/address/logic/commands/AppointmentRemarkCommand.java]: change person to appointment

File: src/main/java/seedu/address/logic/commands/AppointmentRemarkCommand.java [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+public class AppointmentRemarkCommand extends Command {
+    public static final String COMMAND_WORD = "remark_a";
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Edits the remark of the appointment identified "
+            + "by the index number used in the last appointment listing. "
+            + "Existing remark will be overwritten by the input.\n"
+            + "Parameters: INDEX (must be a positive integer) "
+            + "r\\ [REMARK]\n"
+            + "Example: " + COMMAND_WORD + " 1 "
+            + "r/ Patient to follow up in 1 month.";
```
Comment (derekjxtan) [inline src/main/java/seedu/address/logic/commands/AppointmentRemarkCommand.java:28]: delimiter should be backslash instead. use the constant `PREFIX_REMARK`

**Review by yanghengtang — COMMENTED** [review]
only javadoc edits to make

**Review by yanghengtang — APPROVED** [review]
lgtm

**Pull request merged**